src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <container class="container">
+      <div class="stars--0">
+        <div class="darkstars"></div>
+        <div class="darkstars"></div>
+        <div class="darkstars"></div>
+        <div class="darkstars"></div>
+        <div class="darkstars"></div>
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
fix prev comments about task requirements.
no need make different classes for active or non-active star. use nth-child and stars--0, stars--1...stars--5 modifier.
set `stars_star` class for each star and `stars` for each stars row.